### ISSUE TITLE
[PROPOSAL] Expanding Pylint in opensearchpy package

### ISSUE THREAD

**Issue by macohen** [opensearch-project/opensearch-py#647]
### What are you proposing?
In working on expanding the number of lints used in nox (#643, #645), I enabled only four lints for the opensearchpy folder in noxfile.py just to make it simpler to add the lints for other cases. Once the PR (#643) is merged with opensearchpy disabled, if someone can pick up enabling opensearchpy as a separate task to clean up, that would be good.

### What users have asked for this feature?
#610 

### What problems are you trying to solve?
Linting can reduce burden on code reviewers by normalizing code standards through code itself.

### What is the developer experience going to be?
* In noxfile.py, there is a function called lint_per_folder which has the settings for this work. 

1. Follow the instructions for running `nox -rs lint`
2. updating the code according to the rules until no more messages appear. 
3. Add a new lint from #610, and then go to step 1 until you're done or questioning your own sanity

#### Are there any security considerations? 
None

#### Are there any breaking changes to the API
No

### What is the user experience going to be?
N/A

### Why should it be built? Any reason not to?
If the lints do nothing for the developer experience, this doesn't matter/

**Comment by macohen** [comment]
closing this. opensearchpy is enabled through the .pylintrc files. Thanks!
